spec/features/delete_user_spec.rb

```diff
@@ -0,0 +1,24 @@
+require_relative '../spec_helper'
+
+RSpec.describe 'Delete user', type: :feature do
+  context 'When on Main Page' do
+    before(:each) do
+      page.windows[0].maximize
+      visit('/')
+      click_link('войдите в личный кабинет')
+      click_link('Зарегистрироваться')
```

Comment:
You can move code related to user creation to a separate method and also use it in other specs (for example in create one)
NOTE: make sure you do not use other general steps like opening page or login in register_user method

Reply:
I think these steps are general and independent from each other and I don't know in which single method I should use them.